client/example_container_put_test.go

```diff
@@ -0,0 +1,143 @@
+package client_test
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/nspcc-dev/neo-go/pkg/crypto/keys"
+	netmapv2 "github.com/nspcc-dev/neofs-api-go/v2/netmap"
+	"github.com/nspcc-dev/neofs-sdk-go/client"
+	"github.com/nspcc-dev/neofs-sdk-go/container"
+	"github.com/nspcc-dev/neofs-sdk-go/container/acl"
+	cid "github.com/nspcc-dev/neofs-sdk-go/container/id"
+	"github.com/nspcc-dev/neofs-sdk-go/netmap"
+	"github.com/nspcc-dev/neofs-sdk-go/user"
+)
+
+func getPrivateKey() *keys.PrivateKey {
+	// account took from https://github.com/nspcc-dev/neofs-aio
+	key, err := keys.NEP2Decrypt("6PYM8VdX2BSm7BSXKzV4Fz6S3R9cDLLWNrD9nMjxW352jEv3fsC8N3wNLY", "one", keys.NEP2ScryptParams())
+	if err != nil {
+		panic(err)
+	}
+
+	return key
+}
+
+func getAccountID() user.ID {
+	var acc user.ID
+
+	key := getPrivateKey()
+	user.IDFromKey(&acc, key.PrivateKey.PublicKey)
+
+	return acc
+}
+
+func neofsClient() (*client.Client, error) {
+	key := getPrivateKey()
+
+	var prmInit client.PrmInit
+
+	prmInit.SetDefaultPrivateKey(key.PrivateKey) // private key for request signing
+	prmInit.ResolveNeoFSFailures()               // enable erroneous status parsing
+
+	var c client.Client
+	c.Init(prmInit)
+
+	var prmDial client.PrmDial
+	prmDial.SetServerURI("grpc://localhost:8080") // endpoint address
+	prmDial.SetTimeout(15 * time.Second)
+	prmDial.SetStreamTimeout(15 * time.Second)
+
+	if err := c.Dial(prmDial); err != nil {
+		return nil, fmt.Errorf("dial %v", err)
+	}
+
+	return &c, nil
+}
+
+func ExampleClient_ContainerPut() {
+	ctx := context.Background()
+	accountID := getAccountID()
+
+	c, err := neofsClient()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	containerID, err := createContainer(ctx, c, accountID, "name-1")
```

Comment:
I'm not sure these internal functions will be included into examples shown on pkg.go.dev, like https://pkg.go.dev/github.com/nspcc-dev/neo-go@v0.101.1/pkg/rpcclient/invoker#example-Invoker.

Reply:
Moved to single function